config/sync/google_map_field.settings.yml

```diff
@@ -0,0 +1,5 @@
+_core:
+  default_config_hash: mWL2dHvi9X6xknPeaUo2dSbC4IGZ-o-bdn2kRs0MzII
+google_map_field_auth_method: 1
+google_map_field_apikey: ''
```

Comment:
Do we need a settings.php config override so we can set the value from an env var?

Reply:
That's true.